lexis/render/colorMap.fbs

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2015-2016, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Cyrille Favreau <cyrille.favreau@epfl.ch>
+ *                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+ *                     Juan Hernando <juan.hernando@epfl.ch>
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+namespace lexis.render.detail;
+
+// Control point
+table ControlPoint
+{
+    x: float;
+    y: float;
+}
+
+// 1 channel transfer function
+table TransferFunction
+{
+    attribute: string;
+    points: [ControlPoint];
+}
+
+// 4 channel transfer function
+table ColorMap
+{
+    r: [ControlPoint];
+    g: [ControlPoint];
+    b: [ControlPoint];
+    a: [ControlPoint];
+}
+
+// Range of values for which the transfer function applies
```

Comment:
Refresh my memory, what is the rendering engine supposed to do with data values that are outside the range? They become fully transparent or was it something different?